db/schema.rb

```diff
@@ -119,6 +119,13 @@
   add_index "plagiarism_match_links", ["other_task_id"], name: "index_plagiarism_match_links_on_other_task_id", using: :btree
   add_index "plagiarism_match_links", ["task_id"], name: "index_plagiarism_match_links_on_task_id", using: :btree
 
+  create_table "project_convenors", force: true do |t|
```

Comment:
Something seems wrong here... the database schema shouldn't have changed for this.